### PR TITLE
Document copy/translation/i18n/PO file process

### LINKED ISSUE #1446: Documentation: Copy process README.md
- Document the Copy process between Product/Design/Engineering
- Put the docs in `https://github.com/cal-itp/benefits/tree/dev/benefits/locale` for now
- The most crucial thing is that this page has the link to the Benefits Configurable Strings spreadsheet, which is the sheet that is linked to the Figma designs

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes #1446 

## What this PR does
- Non-technical doc to document how we divide the responsibilities (Product, Design, Engineering) of getting application copy drafted, edited, approved, designed into Figma, translated, and finally, into the application in English and Spanish.
- Most importantly, this page will have the link to _the_ spreadsheet, that holds the English and Spanish translations -- so we don't have to get lost in Google Drive or various GitHub issues to find it.
- This PR also updates Engineering's i18n docs with our new processes

## Process
- [x] @machikoyasuda to start off the doc file
- [x] @machikoyasuda to move the file to /docs/
- [x] @machikoyasuda to hand-off iBabbleOn process to @indexing 
- [ ] @indexing to review

### CODE REVIEW

**Comment by machikoyasuda** [comment]
@indexing I started the Markdown file. 

Please propose changes by making a GitHub comment on the line (or adding a suggestion commit https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-comments-to-a-pull-request), and I can merge the suggestions in.

File: benefits/locale/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Translation strings include all application copy, including:
+
+- All application copy
+- Image and illustration alt tags
+- Page titles used in the browser tab
+- In-line link URLs
+
+## BENEFITS STRING SPREADSHEET NAME TBD
+
+The human-readable version of the English and Spanish translation strings for the application are delivered to Design and Engineering by Product, and live at this link: [Benefits String Spreadsheet NAME TBD](https://docs.google.com/spreadsheets/d/13YZLv7wf8dAk_HnEP-KLEUurTggx8QOx9HovEW4_UjI/edit#gid=0).
```
Comment (machikoyasuda) [inline benefits/locale/README.md:14]:
```suggestion
The human-readable version of the English and Spanish translation strings for the application are delivered to Design and Engineering by Product, and live at this link: [Benefits Copy Spreadsheet](https://docs.google.com/spreadsheets/d/13YZLv7wf8dAk_HnEP-KLEUurTggx8QOx9HovEW4_UjI/edit#gid=0).
```

**Review by thekaveman — CHANGES_REQUESTED** [review]
These docs are super helpful! 

Can I suggest/request that we move them into a page in our `/docs` folder so they show up in the https://docs.calitp.org/benefits site and are more easily accessible for everyone? Maybe under the [Configuration section](https://docs.calitp.org/benefits/configuration/)?

We can still use the README in `/benefits/locale` to link to this online documentation (see an example of this pattern in the [`/terraform` directory](https://github.com/cal-itp/benefits/blob/dev/terraform/README.md))

File: benefits/locale/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- Compiling copy in [Benefits String Spreadsheet NAME TBD](https://docs.google.com/spreadsheets/d/13YZLv7wf8dAk_HnEP-KLEUurTggx8QOx9HovEW4_UjI/edit#gid=0), ready to be used by Design, so Design can sync the spreadsheet to Figma.
+- Engaging with the translation agency, [iBabbleOn](https://ibabbleon.com/), to get Spanish translations ready for Engineering.
+- Ensure English and Spanish copy is ready for Engineering.
+
+### Design
+
+- Syncing copy from [Benefits String Spreadsheet](https://docs.google.com/spreadsheets/d/13YZLv7wf8dAk_HnEP-KLEUurTggx8QOx9HovEW4_UjI/edit#gid=0) into Figma.
+- Ensuring the string is in the appropriate column (e.g. `Subtitle`, `ButtonLabel`)
+
+### Engineering
```
Comment (thekaveman) [inline benefits/locale/README.md:33]: Need to add a reminder to run `bin/makemessages.sh`: https://github.com/cal-itp/benefits/pull/1436#discussion_r1242647512

**Review by thekaveman — CHANGES_REQUESTED** [review]

**Comment by machikoyasuda** [comment]
## Current status
- @indexing and @machikoyasuda met today to hand-off iBabbleOn process to @indexing 
- Once Andy re-formats the Google Spreadsheet to fit the translation needs, sends the request, receives the requested translations and hands the translations back to Dev, he can update the documentation for full deatils
- Discussed how and where the new variables should be kept. Andy decided not to worry about getting the new variables ready and translated in this round. The variables can be written and translated in the next next round. I believe we have all the variable data for MST, SacRT, Veterans **except** for the `[card functionality]`.. but I'm not 100% sure.

```
- [affected service]: public transportation or light rail or bus service
- [action]: reduce
- [rate of discount]: 50%
- [transit type]: fixed routes
- [card functionality]: tap this card when you get on and off the bus
```

**Comment by machikoyasuda** [comment]
Lots of new stuff here: https://github.com/cal-itp/benefits/issues/1410#issuecomment-1616237332

**Comment by machikoyasuda** [comment]
NIST is `el Instituto Nacional de Estándares y Tecnología`
Source: https://login.gov/es/who-uses-login/
https://github.com/18F/identity-site/blob/50e1724eaf00dba130375886b9d683a71c8d8b29/content/_landing/who_uses_login._es.md?plain=1#L24

File: benefits/locale/README.md (hunk 1 above)
Comment (thekaveman) [inline benefits/locale/README.md:14]: Please update document URL: https://docs.google.com/spreadsheets/d/1_Gi_YbJr4ZuXCOsnOWaewvHqUO1nC1nKqiVDHvw0118

File: benefits/locale/README.md (hunk 2 above)
Comment (thekaveman) [reply benefits/locale/README.md:33]: Now that #1571 is done, engineers should be editing copy directly in the app for English, then running `bin/makemessages.sh`, then updating the corresponding Spanish in the PO file.

Comment (machikoyasuda) [reply benefits/locale/README.md]: Updated in the dev i18n docs: https://github.com/cal-itp/benefits/pull/1447/files#diff-739a6fd0a2d16786a1fad0499fde882753fee93043bb1f8b1a4b03ba5c75ea06

**Comment by machikoyasuda** [comment]
@indexing This page is ready for your review: https://github.com/cal-itp/benefits/pull/1447/files#diff-def4742fd8f91b94db56a5c970797fe316f277c2d0e2604c0e3ac1bd2c2edb8d

**Comment by machikoyasuda** [comment]
@thekaveman @angela-tran Please review the updated i18n dev docs https://github.com/cal-itp/benefits/pull/1447/files#diff-739a6fd0a2d16786a1fad0499fde882753fee93043bb1f8b1a4b03ba5c75ea06

I tried covering all the new processes, but let me know if I'm missing anything!

File: docs/product-and-design/copy-delivery.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+- Ensure English and Spanish copy is ready for Engineering.
+
+### Design
+
+- Sync copy from [Cal-ITP Benefits Application Copy (Configurable Strings)](https://docs.google.com/spreadsheets/d/1_Gi_YbJr4ZuXCOsnOWaewvHqUO1nC1nKqiVDHvw0118/edit#gid=0) into Figma.
+- Ensure the string is in the appropriate column (e.g. `Subtitle`, `ButtonLabel`)
+
+### Engineering
+
+- Use copy from the English and Spanish language tabs of the spreadsheet, and turning it into code in `django.po` message files. Developer-specific instructions in the [Django message files technical documentation](https://docs.calitp.org/benefits/development/i18n/).
```
Comment (thekaveman) [inline docs/product-and-design/copy-delivery.md:42]:
```suggestion
- Use copy from the English and Spanish language tabs of the spreadsheet, and turning it into code in `django.po` message files. Developer-specific instructions in the [Django message files technical documentation](../development/i18n.md).
```

Comment (thekaveman) [reply docs/product-and-design/copy-delivery.md:42]: We can (should) just use the local file link for pages in our docs site -- this way the links will work e.g. locally (and keep you on the local site), and it works against pages that are being created / don't exist on prod yet.

**Review by thekaveman — CHANGES_REQUESTED** [review]
Minor nit about an internal link, but otherwise this is great and I'd say let's get it published!

**Review by thekaveman — APPROVED** [review]

**Pull request merged**